### PR TITLE
Moving bdutil docs to GitHub

### PR DESCRIPTION
**Pull request by james-malone**
Moving the bdutil docs from cloud.google.com/hadoop to the GitHub repo
so they can live alongside bdutil.

(squashed commits)
Fixes for typos and other issues in the first pass.

Fixes, round 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+The script implements the following commands, which are very similar:
+
+* `bdutil create` creates and starts instances, but will not apply most configuration settings. You can call `bdutil run_command_steps` on instances afterward to apply configuration settings to them. Typically you wouldn't use this, but would use `bdutil deploy` instead.
+* `bdutil deploy` creates and starts instances with all the configuration options specified in the command line and any included configuration scripts.
+
+## Components installed
+
+The latest release of bdutil is `1.3.5`. This bdutil release installs the following versions of open source components:
+
+* Apache Hadoop 2 - 2.7.1
```
Comment (pmkc) [inline README.md]: As far as I recall we never actually defaulted to Hadoop 2. This requires passing -e hadoop2

File: docs/MONITORING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Monitoring jobs
+
+You can monitor the activity of your cluster using the Hadoop web interface while `wordcount` is running. Apache Hadoop provides web interfaces for the Hadoop Distributed File System (HDFS), MapReduce, and YARN. However, because these interfaces do not require user authentication to connect to them, they are insecure. Instead of opening these ports to public traffic, we recommend that you create a secure SSH tunnel from your local network to your [Compute Engine network](https://cloud.google.com/compute/docs/networking).
+
+## Set up a secure SSH tunnel
+
```
Comment (pmkc) [inline docs/MONITORING.md:6]: Use `bdutil socksproxy` instead

**Review by pmkc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (james-malone) [reply README.md]: Fixed!

File: docs/MONITORING.md (hunk 2 above)
Comment (james-malone) [reply docs/MONITORING.md:6]: Fixed!

**Pull request merged**